src/generated/stellar-xdr_generated.js

```diff
@@ -6,3656 +6,3549 @@
-//
-// ===========================================================================
-xdr.union("AllowTrustOpAsset", {
-  switchOn: xdr.lookup("AssetType"),
-  switchName: "type",
-  switches: [
-    ["assetTypeCreditAlphanum4", "assetCode4"],
-    ["assetTypeCreditAlphanum12", "assetCode12"],
-  ],
-  arms: {
-    assetCode4: xdr.opaque(4),
-    assetCode12: xdr.opaque(12),
-  },
-});
-
-// === xdr source ============================================================
-//
-//   struct AllowTrustOp
-//   {
-//       AccountID trustor;
-//       union switch (AssetType type)
-//       {
-//       // ASSET_TYPE_NATIVE is not allowed
-//       case ASSET_TYPE_CREDIT_ALPHANUM4:
-//           opaque assetCode4[4];
-//   
-//       case ASSET_TYPE_CREDIT_ALPHANUM12:
-//           opaque assetCode12[12];
-//   
-//           // add other asset types here in the future
-//       }
-//       asset;
-//   
-//       bool authorize;
-//   };
-//
-// ===========================================================================
-xdr.struct("AllowTrustOp", [
-  ["trustor", xdr.lookup("AccountId")],
-  ["asset", xdr.lookup("AllowTrustOpAsset")],
-  ["authorize", xdr.bool()],
-]);
-
-// === xdr source ============================================================
-//
-//   struct ManageDataOp
-//   {
-//       string64 dataName;
-//       DataValue* dataValue; // set to null to clear
-//   };
-//
-// ===========================================================================
-xdr.struct("ManageDataOp", [
-  ["dataName", xdr.lookup("String64")],
-  ["dataValue", xdr.option(xdr.lookup("DataValue"))],
-]);
-
-// === xdr source ============================================================
-//
-//   struct BumpSequenceOp
-//   {
-//       SequenceNumber bumpTo;
-//   };
-//
-// ===========================================================================
-xdr.struct("BumpSequenceOp", [
-  ["bumpTo", xdr.lookup("SequenceNumber")],
-]);
-
-// === xdr source ============================================================
-//
-//   union switch (OperationType type)
-//       {
-//       case CREATE_ACCOUNT:
-//           CreateAccountOp createAccountOp;
-//       case PAYMENT:
-//           PaymentOp paymentOp;
-//       case PATH_PAYMENT:
-//           PathPaymentOp pathPaymentOp;
-//       case MANAGE_OFFER:
-//           ManageOfferOp manageOfferOp;
-//       case CREATE_PASSIVE_OFFER:
-//           CreatePassiveOfferOp createPassiveOfferOp;
-//       case SET_OPTIONS:
-//           SetOptionsOp setOptionsOp;
-//       case CHANGE_TRUST:
-//           ChangeTrustOp changeTrustOp;
-//       case ALLOW_TRUST:
-//           AllowTrustOp allowTrustOp;
-//       case ACCOUNT_MERGE:
-//           AccountID destination;
-//       case INFLATION:
-//           void;
-//       case MANAGE_DATA:
-//           ManageDataOp manageDataOp;
-//       case BUMP_SEQUENCE:
-//           BumpSequenceOp bumpSequenceOp;
-//       }
-//
-// ============================================================
```

Comment:
This is autogenerated file and should be ignored. Please revert it to the previous version.

Reply:
Should be easy to have the linters ignore `src/generated`!